src/message-store-sql.ts

```diff
@@ -88,7 +109,7 @@ export class MessageStoreSql implements MessageStore {
   async put(
```

Comment:
Sanity, probably extends to level implementation too: we should also delete the tags data when `delete()` is called right?

Reply:
Because the deletes are cascading, a delete of the message deletes all of the tags + values associated.
In level, each index is independently deleted.

However this instantly made me think of something i missed in `dwn-sdk-js`, which is that we don't need to index the tags of a record that is not `latestBaseState`.

https://github.com/TBD54566975/dwn-sdk-js/pull/725